docs/content/references/sui-api/graphql.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Overview
+description: GraphQL Reference page overview
+---
+
+The Sui GraphQL RPC is a public good that enables interacting with the Sui [network](https://sui.io/networkinfo).
```

Comment:
```suggestion
The Sui GraphQL RPC is a public service that enables interacting with the Sui [network](https://sui.io/networkinfo).
```
Can we use `service`? 'good' I think has too many opportunities to be misunderstood by esl folks.